provider/README.md

```diff
@@ -344,13 +335,48 @@ When multiple query parameters are specified, they should all apply to the retur
 
 ## Realtime Data
 
+### GBFS
+
 All MDS compatible `provider` APIs must expose a public [GBFS](https://github.com/NABSA/gbfs) feed as well. Given that GBFS hasn't fully [evolved to support dockless mobility](https://github.com/NABSA/gbfs/pull/92) yet, we follow the current guidelines in making bike information avaliable to the public. 
 
   - `gbfs.json` is always required and must contain a `feeds` property that lists all published feeds
   - `system_information.json` is always required
   - `free_bike_status.json` is required for MDS
   - `station_information.json` and `station_status.json` don't apply for MDS
 
+### Events
+
+The events endpoint is a near-ish real time feed of events, designed to give access to as recent as possible series of events from the vehicle feed. It functions similarly to status changes, but shall not included data older than 2 weeks (that should live in `/status_changes.`)
+
+Unless stated otherwise by the municipality, this endpoint must return only those events with a `event_location` that [intersects](#intersection-operation) with the [municipality boundary](#municipality-boundary).
+
+> Note: As a result of this definition, consumers should query the [trips endpoint](#trips) to infer when vehicles enter or leave the municipality boundary.
+
+The datatype and scheme are the same as `status_changes
```

Comment:
Looks like you're missing a closing ` here

(also not sure whether it's "scheme" or "schema" here -- I defer to you!)